src/main/io/beeper.c

```diff
@@ -409,8 +410,19 @@ void beeperUpdate(timeUs_t currentTimeUs)
             || (currentBeeperEntry->mode == BEEPER_RX_LOST && !(beeperConfig()->dshotBeaconOffFlags & BEEPER_GET_FLAG(BEEPER_RX_LOST))))) {
 
             if ((currentTimeUs - getLastDisarmTimeUs() > DSHOT_BEACON_GUARD_DELAY_US) && !isTryingToArm()) {
+                static uint8_t dshotBeaconCycle = 1;
                 lastDshotBeaconCommandTimeUs = currentTimeUs;
-                pwmWriteDshotCommand(ALL_MOTORS, getMotorCount(), beeperConfig()->dshotBeaconTone, false);
+                if (beeperConfig()->dshotBeaconTone == DSHOT_BEEP_CYCLE1)
```

Comment:
Please read https://github.com/betaflight/betaflight/blob/master/docs/development/CodingStyle.md and format your code accordingly.